packages/esm-patient-banner-app/src/banner/patient-banner.component.tsx

```diff
@@ -9,25 +9,43 @@ import CustomOverflowMenuComponent from '../ui-components/overflow-menu.componen
 import styles from './patient-banner.scss';
 import { useTranslation } from 'react-i18next';
 import { Button } from 'carbon-components-react';
-import { ExtensionSlot, age } from '@openmrs/esm-framework';
+import { ExtensionSlot, age, navigate } from '@openmrs/esm-framework';
 
 interface PatientBannerProps {
   patient: fhir.Patient;
   patientUuid: string;
+  patientUrl?: string;
+  hidePatientSearchPanel?: () => void;
 }
 
-const PatientBanner: React.FC<PatientBannerProps> = ({ patient, patientUuid }) => {
+const PatientBanner: React.FC<PatientBannerProps> = ({ patient, patientUuid, patientUrl, hidePatientSearchPanel }) => {
   const { t } = useTranslation();
   const state = React.useMemo(() => ({ patientUuid }), [patientUuid]);
   const [showContactDetails, setShowContactDetails] = React.useState(false);
   const toggleContactDetails = React.useCallback(() => setShowContactDetails((value) => !value), []);
 
+  const patientAvatar = (
+    <div className={styles.patientAvatar} role="img">
+      <ExtensionSlot extensionSlotName="patient-photo-slot" state={state} />
+    </div>
+  );
   return (
     <div className={styles.container} role="banner">
       <div className={styles.patientBanner}>
-        <div className={styles.patientAvatar} role="img">
-          <ExtensionSlot extensionSlotName="patient-photo-slot" state={state} />
-        </div>
+        {patientUrl ? (
+          <div
```

Comment:
Never use `<div>` tags to make clickable elements. Use `<a>` if it's a link with an `href`, or `<button>` if it might do other stuff with `onClick`. Here, you want a `<button>`.

Reply:
Resolved